frontend/src/container/OnboardingContainer/Steps/DataSource/DataSource.styles.scss

```diff
@@ -41,3 +41,19 @@ div[class*='-setup-instructions-container'] {
 .service-name-container {
 	width: 100%;
 }
+
+.intgeration-page-container {
+	background-color: var(--bg-ink-400);
+	border-color: var(--bg-slate-500);
+}
+
```

Comment:
do we need the same styles for the light mode ?

Reply:
Yes we are giving the same styles and As discussed with @GeekBoySupreme We're not actively prioritizing light mode.

Reply:
cool just make sure that the box is readable enough in light mode